mkosi.credentials/userdb.user.testuser

```diff
@@ -0,0 +1,16 @@
+{
+    "userName": "testuser",
+    "homeDirectory": "/home/testuser",
```

Comment:
maybe drop this again now that the bug around that is fixed.

Reply:
But the fallback home directory is `/`, not `/home/testuser` no?

Reply:
Nvm, found the auto stuff